backend/src/v5/models/revisions.js

```diff
@@ -25,13 +28,13 @@ const Revisions = {};
 const excludeVoids = { void: { $ne: true } };
 const excludeIncomplete = { incomplete: { $exists: false } };
 
-const collectionName = (model) => `${model}.history`;
+const collectionName = (modelType, model) => (modelType === modelTypes.DRAWING ? DRAWINGS_HISTORY_COL : `${model}.history`);
 
-const findRevisionsByQuery = (teamspace, model, query, projection, sort) => db.find(teamspace,
-	collectionName(model), query, projection, sort);
+const findRevisionsByQuery = (teamspace, model, modelType, query, projection, sort) => db.find(teamspace,
```

Comment:
Can we review all these functions....

Drawings version will not work well if we don't pass model id into the query.

Reply:
These should start working a bit like custom tickets functions where we pass in the project/model etc into the query if the model type is drawings